distributed/utils.py

```diff
@@ -611,7 +611,7 @@ def read_block(f, offset, length, delimiter=None):
 
     Parameters
     ----------
-    fn: string
+    f: string
```

Comment:
`f` is commonly used for a file while `fn` is commonly used for a filename.  If the intent is for this to be a filename then we might instead change f->fn everywhere.  If this is commonly used for non-file paths then we might instead use the term `path` everywhere.

@martindurant any preferences?